devel/cmake-devel/Portfile

```diff
@@ -80,13 +80,14 @@ patchfiles-append \
     patch-remove-homebrew-references.diff
 
 depends_lib-append \
+    path:lib/libuv.dylib:libuv \
     port:curl \
     port:expat \
     port:zlib \
     port:bzip2 \
     port:libarchive \
-    port:ncurses \
-    port:libuv
+    port:ncurses
+
```

Comment:
Maybe do not add an extra empty line here?

Reply:
> Maybe do not add an extra empty line here?

Agreed